docs/restful-interface.rst

```diff
@@ -1,499 +1,400 @@
-.. _restful-interface:
+************************************
+Querying Pfam using the InterPro API 
+************************************
 
-*****************
-RESTful interface
-*****************
+This is an introduction to the `InterPro API <https://www.ebi.ac.uk/interpro/api>`_ to retrieve Pfam annotations. A programmatic interface, 
+commonly called an `Application Programming Interface <http://en.wikipedia.org/wiki/Api>`_ (API) allows users to write scripts or programs 
+to access data, rather than having to rely on a browser to view a site.
 
-This is an introduction to the `"RESTful" <http://www.xfront.com/REST-Web-Services.html>`_ interface to the Pfam website. `REST <http://en.wikipedia.org/wiki/Representational_State_Transfer>`_ (or Representation State Transfer) refers to a style of building websites which makes it easy to interact programmatically with the services provided by the site. A programmatic interface, commonly called an `Application Programming Interface <http://en.wikipedia.org/wiki/Api>`_ (API) allows users to write scripts or programs to access data, rather than having to rely on a browser to view a site.
 
 Basic concepts
 ==============
 
 URLs
 ----
 
-A RESTful service typically sends and receives data over `HTTP <http://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol>`_, the same protocol that's used by websites and browsers. As such, the services provided through a RESTful interface are identified using URLs.
+A RESTful service typically sends and receives data over `HTTP <http://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol>`_, 
+the same protocol that's used by websites and browsers. As such, the services provided through a RESTful interface are identified using URLs.
+
+In the InterPro website we use a different URL to provide the standard HTML representation of Pfam data and the alternative 
+programmatic JSON format through the API. 
+
+To see the data for a particular Pfam-A family, you would visit the following URL in your browser:
+
+  `/entry/pfam/PF02171/ <https://www.ebi.ac.uk/interpro/entry/pfam/PF02171/>`_
+
+To retrieve the data in JSON format, just add an extra parameter, api to the URL:
```

Comment:
extra parameter, `api`, to the URL: ??

Reply:
I should have put `api` in bold, I want to refer to the url that is only different because the api url has the word `api` in it

Reply:
that's what I meant, highligthing this word in bold, markdown or however you consider better